dm/task-configuration-file-full.md

```diff
@@ -103,7 +103,11 @@ mydumpers:                           # dump 处理单元的运行配置参数
 loaders:                             # load 处理单元的运行配置参数
   global:                            # 配置名称
     pool-size: 16                    # load 处理单元并发执行 dump 处理单元的 SQL 文件的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
-    dir: "./dumped_data"             # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。同实例对应的不同任务必须配置不同的目录
+
+    # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。
+    # 支持配置为本地文件系统路径，也支持配置为 Amazon S3 路径，如: s3://dm_bucket/dumped_data?region=us-west-2&endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
+    # 注意：同实例对应的不同任务必须配置不同的目录
```

Comment:
```suggestion
    # 注意：当使用 S3 时同实例对应的不同任务必须配置不同的目录
```

Reply:
This limit will be deleted, because we will add sfffix or subdir from different task and source.